### ISSUE TITLE
Search Results Web Part Does not display Title

### ISSUE THREAD

**Issue by mwsaunders** [SharePoint/sp-dev-solutions#214]
#### Category
- [ ] Question
- [X] Bug
- [ ] Enhancement

#### Expected or Desired Behavior
When a title is added in the web part the title should be displayed in the UI

#### Observed Behavior
No title displayed in UI - I know this has already been reported and was reported as being fixed.

I started out with the Master Branch a couple days ago and just now got the developer branch and installed that. I am seeing the same results and the version 3.4.0.0 did not change from the Master Branch to the Developer Branch.
I am a novice at this stuff.

Regards,

**Comment by wobba** [comment]
Duplicate #190. Verified with current modern-search-dev branch and works as expected. Will be part of the July release coming probably this week (which means merge with master).

**Comment by mwsaunders** [comment]
Hmm,  As I wrote I did get the modern-search-dev branch yesterday.

Should the version have changed?

**Comment by wobba** [comment]
@mwsaunders Not yesterday, bumped it to 3.5 today but that should not matter. I tested on a package I built 5 days ago.

**Comment by mwsaunders** [comment]
Okay - I removed the app from the site, deleted it from the site collection recycle bin and secondary recycle bin, the removed it from the Tenant app catalog and reinstalled it with yesterdays dev branch (still on 3.4.0.0).  Now the title is working for me.

To re-iterate what I had done was to add the master branch to the app catalog, found the title was not working, then put the dev branch in the app catalog - process prompted with app with same name exists, do I want to update and I clicked yes.  Lesson Learned.

Thanks to all for their work on the Mondern Search Solution.
